ci/build-parasail.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Download a fixed version of Parasail, compile it, and install to /usr/local.
+# Mainly used to build Linux wheels, inside an appropriate manylinux container.
```

Comment:
if this is only used on linux, can't we get compiled versions of `parasail` from a linux package manager? Are we building parasail ourselves because it helps us build manylinux wheels?